package.json

```diff
@@ -44,4 +45,4 @@
       "packages/teleterm"
     ]
   }
-}
+}
```

Comment:
Missing new line at the end.

Reply:
Weird for some reason my vscode keeps stripping it. readded anyway.